src/output-json-flow.c

```diff
@@ -221,6 +221,11 @@ void EveAddFlow(Flow *f, JsonBuilder *js)
         jb_set_uint(js, "bytes_toclient", f->tosrcbytecnt);
     }
 
+    TcpSession *ssn = (TcpSession *)f->protoctx;
```

Comment:
I'd like to see this wrapped in `if (f->proto == IPPROTO_TCP) { ... }`. We don't currently use `f->protoctx` for other things, but we could.

Reply:
ping @rtkjweeks ?